package.json

```diff
@@ -16,12 +16,14 @@
 		"@astrojs/sitemap": "^1.0.0",
 		"@emotion/styled": "^11.10.5",
 		"@fortawesome/fontawesome-svg-core": "^6.2.1",
-		"@fortawesome/free-brands-svg-icons": "^6.2.1",
+		"@fortawesome/free-brands-svg-icons": "^6.4.0",
+		"@fortawesome/free-regular-svg-icons": "^6.4.0",
 		"@fortawesome/react-fontawesome": "^0.2.0",
 		"@mui/material": "^5.11.2",
 		"@nanostores/react": "^0.4.1",
 		"@types/fs-extra": "^11.0.1",
 		"astro": "^1.7.2",
+		"font-awesome": "^4.7.0",
```

Comment:
Why does this need to be added?

Reply:
emilien wants his website instead, so i needed regular icons

Reply:
No, we're only putting Instagram and LinkedIn

Reply:
Can be removed

Reply:
?